pygeppetto/model/utils/pointer_utility.py

```diff
@@ -0,0 +1,347 @@
+from ..exceptions import GeppettoModelException
```

Comment:
I know it's generated code... but it's really non pythonic and the logic could be improved sometimes :p

Reply:
Agree. I don't like many for loops in particular. It may need time though to make everything pythonic. And we have a lot of things to port there :/

Reply:
Ok, if you have a lot of things to port, then the best thing is to use automatic translation as you did and non pythonic code for a first run, as long as tests are here. We could refactor everything later.

I was very slow when I started everything because I was rewriting everything (removing some non-javaic code also from the Java version :p). To my opinion, it's the best thing to do, but it require extra-time and a lot of hands to do it properly.

Reply:
I like the approach to make some refactoring before merging. I'll also keep in mind to do small pythonic refactorings whenever possible also during the porting

Reply:
I think it's much more Pythonic now. Still we have some ugly loops but I wouldn't go in the depth of complex algorithms just for the sake of the style.
I kept the camelCase: I'm not yet convinced about the snake_case at any cost; it's not only saving time during the porting but also keeping the correspondence with Java and Javascript where it comes to API methods, which has a great value in my opinion. What about a style guide deriving from PEP-8 with camel case encouraged only where there is a correspondence with the Java/Javascript side?